sdk/azidentity/environment_credential.go

```diff
@@ -4,27 +4,40 @@
 package azidentity
 
 import (
+	"fmt"
 	"os"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
 )
 
 // NewEnvironmentCredential creates an instance of the EnvironmentCredential type and reads client secret details from environment variables.
 // If the expected environment variables are not found at this time, the GetToken method will return the default AccessToken when invoked.
```

Comment:
Unrelated to logging, but is this part of the comment actually true?  Looks like if the environment vars aren't found there will be no future call to GetToken().

Reply:
You're right, this was the original comment. I will remove it.